hub/powertoys/file-explorer.md

```diff
@@ -1,27 +1,32 @@
 ---
-title: PowerToys File Explorer utility for Windows
-description: A File Explorer add on that enables Markdown and SVG previews
-ms.date: 05/28/2021
+title: PowerToys File Explorer add-ons utility for Windows
+description: A File Explorer add on that enables different preview pane and thumbnail renderers for different file types.
+ms.date: 01/23/2022
 ms.topic: article
 ms.localizationpriority: medium
-no-loc: [PowerToys, Windows, File Explorer]
+no-loc: [PowerToys, Windows, File Explorer, Monaco]
 ---
 
 # File Explorer add-ons utility
 
 File Explorer add-ons currently include:
 
-- Preview Pane rendering of SVG icons (.svg)
-- Preview Pane rendering of Markdown files (.md)
-- Preview Pane rendering of PDF files (.pdf)
-- Icon thumb preview for SVG icons
-- Icon thumb preview for PDF files
+- Preview Pane rendering for
+  - SVG icons (.svg)
+  - Markdown files (.md)
+  - source code files (.cs, .cpp, .rs, ...)
```

Comment:
```suggestion
  - Source code files (.cs, .cpp, .rs, ...)
```

Reply:
I don't agree because the sentence is `Preview Pane rendering for source code files (.cs, .cpp, .rs, ...)` and the s shouldn't be big then

Reply:
Let's remove the word `for` and use `:` instead. Looks better.